commons/passive/garmin/garmin_generic_device_info.avsc

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "GarminGenericDeviceInfo",
+  "type": "record",
+  "namespace": "org.radarcns.passive.garmin",
+  "doc":"Device information for Garmin devices.",
+  "fields": [
+
+   {
+              "name":"time",
+              "type":"double",
+              "doc":"Last updated time."
+           },
+           {
+              "name":"timeReceived",
+              "type":"double",
+              "doc":"Device receiver timestamp in UTC (s)."
+           },
+
+    {
+      "name": "macAddress",
+      "type": "string",
+       "doc":"MAC address as received from Garmin device."
+    },
+    {
+      "name": "connectionState",
+      "type": "string",
+       "doc":"Connection state values. Possible states - DISCONNECTED, CONNECTING, CONNECTED."
```

Comment:
Please use an enum, with `UNKNOWN` as first symbol and `UNKNOWN` as default.

Reply:
Done. will be available in latest schema.